queries.sql

```diff
@@ -150,4 +150,4 @@ WHERE animals.name = (
   WHERE vet = 'Maisy Smith'
   GROUP BY animal
```

Comment:
- [ ] It would be better if we change the “TEXT” to “int” so that we can properly link the tables. For example;  `CREATE TABLE specializations(
 vet TEXT,
 specie TEXT
);`
Kindly check the visits table as well

Reply:
![Screenshot 2022-09-16 232852](https://user-images.githubusercontent.com/92556803/190825158-d03e0439-b4a4-41a0-b1ef-8ee2f42548f7.png)
i get this error whenever i try to change it to int

Reply:
@addod19

Reply:
and vet is also not an integer it is a string